### DISCUSSION TITLE
Magic question : How much tags are supported for textcat multilabel

### DISCUSSION THREAD

**Discussion by warstrolo** [explosion/spaCy#10169] [Help: Other Questions] [answered]
Hi everyone, 
How are you all ? 

I have this big question, and I did not find an answer : 
- How much tags can we have at maximum in this pipe ? 

Cause I will have to add like billions of tags to my datasets
So idk if it's support it 

Thanks for your time :grin:

**Comment by polm** [comment, marked as answer]
There is no built-in limit on the number of labels. 

With a lot of labels, you're likely to run into issues with not having enough data, or into memory limitations. If you have issues using lots of labels that seem like implementation problems rather than data/memory problems let us know.
